docs/doxybook_config.json

```
@@ -0,0 +1,49 @@
+{
+  "baseUrl": "/api/",
```

I think this value is missing the `/thrust` prefix needed for deployment. Currently, pages like https://nvidia.github.io/thrust/api/groups/group__algorithms.html have links to https://nvidia.github.io/api/groups/group__copying.html, which is invalid because it's missing the `/thrust` prefix.

See the note here: https://github.com/matusnovak/doxybook2#config-examples

I'm not sure if it is possible to make both local and GitHub Pages builds work with the same configuration file.

```suggestion
  "baseUrl": "/thrust/api/",
```

---

We should be able to fix this pretty easily. Instead of hard coding it in the config file, we can pass it on the command line to doxybook with `--config-data`.

---

Fixed in `fdf106b6`. `generate_markdown.bash` now defaults to setting `baseUrl` to `/thrust/api/`, and has a `--local` flag to set it to `/api/` instead.